cmdb.js

```diff
@@ -39,7 +40,7 @@ cmdb.prototype._fetch = function _fetch(locals, path, method, body) {
 	}
 
 	// HACK: CMDB decodes paths before they hit its router, so do an extra encode on the whole path here
-	path = encodeURIComponent(path);
+//	path = encodeURIComponent(path);
```

Comment:
I'd be interested to know what changed so that this is no longer needed.  (It's good that it's not needed, just would like to understand it)